src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax;
+
+import core.basesyntax.exception.RegistrationDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void createValidUser() {
+        user = new User();
+        user.setLogin("validLogin");
+        user.setPassword("validPassword");
+        user.setAge(33);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        Assertions.assertThrows(RegistrationDataException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void register_nullUserLogin_notOk() {
+        user.setLogin(null);
+        Assertions.assertThrows(RegistrationDataException.class, () -> {
+            registrationService.register(user);
+        });
```

Comment:
Modifying the static user instance directly affects other tests since it's shared. To maintain independence between tests, create a new User instance within each test method where you need a modified user.